### PR TITLE
vm/permissions

### PR DESCRIPTION
**Pull request by harshshah-tenable**
- doc string update and resp update.

### CODE REVIEW

File: tenable/io/v3/vm/permissions/api.py [hunk 1]
```diff
@@ -44,13 +44,13 @@ def change(self, otype: str, id: UUID, *acls: Dict) -> None:
                 `permissions documentation`_ for more details.
```
Comment (ngaglani-tenable) [inline tenable/io/v3/vm/permissions/api.py:44]: Is the `_` here intended?

**Review by ngaglani-tenable — APPROVED** [review]

File: tenable/io/v3/vm/permissions/api.py (hunk 1 above)
Comment (harshshah-tenable) [reply tenable/io/v3/vm/permissions/api.py:44]: yes, it is the syntax required for adding links in documentation.

File: tenable/io/v3/vm/permissions/api.py [hunk 2]
```diff
@@ -44,13 +44,13 @@ def change(self, otype: str, id: UUID, *acls: Dict) -> None:
...
 
                 i.e {'type': 'user', 'id': 2236706, 'permissions': 64}
-                    type:
-                        The type of permission (default, user, group).
-                    id (int):
-                        The unique ID of the user or group.
-                    permissions (int):
-                        The permission value to grant access as described in
-                        `permissions documentation`_.
+                type:
```
Comment (Mangusingh-te) [inline tenable/io/v3/vm/permissions/api.py]: Specify type of **type** param. Eg. **type (str)**.

**Review by Mangusingh-te — CHANGES_REQUESTED** [review]
Apart from minor changes requested, other changes looks good to me.

File: tenable/io/v3/vm/permissions/api.py (hunk 2 above)
Comment (harshshah-tenable) [reply tenable/io/v3/vm/permissions/api.py]: Updated

Comment (alpesh-te) [reply tenable/io/v3/vm/permissions/api.py]: Verified!

**Review by alpesh-te — APPROVED** [review]

**Pull request merged**